packages/balancer/agents/autonomous_fund/aea-config.yaml

```diff
@@ -144,12 +144,12 @@ models:
       max_index_value: 100
       min_index_value: 0
       observation_aggregator_function: median
-      observation_interval: 72000
-      on_chain_service_id: null
+      observation_interval: ${int:72000}
+      on_chain_service_id: ${int:0}
       allowed_lp_addresses: ${list:[]}
       enforce_allowlist: ${bool:true}
       multisend_address: ${str:0xA238CBeb142c10Ef7Ad8442C6D1f9E89e07e7761}
-      pool_tokens: ${SKILL_AUTONOMOUS_FUND_ABCI_MODELS_PARAMS_ARGS_POOL_TOKENS:list:["0x3A78BF1783a0187c1C8000e41C2a008897D0a35f","0xC97b465daC9f52A26F2A234c658a57f5B3f15D19","0xfe46A8F577d3367848bdd127173B7d5F14a6088C"]}
+      pool_tokens: ${list:["0x3A78BF1783a0187c1C8000e41C2a008897D0a35f","0xC97b465daC9f52A26F2A234c658a57f5B3f15D19","0xfe46A8F577d3367848bdd127173B7d5F14a6088C"]}
```

Comment:
Are these wETH, wBTC and USDC?

Reply:
they should be, I will add them now, along with the allowlist.

Reply:
https://etherscan.io/address/0x3A78BF1783a0187c1C8000e41C2a008897D0a35f is not looking right ...

Reply:
https://etherscan.io/address/0xC97b465daC9f52A26F2A234c658a57f5B3f15D19 same

Reply:
https://etherscan.io/address/0xfe46A8F577d3367848bdd127173B7d5F14a6088C same

Reply:
I suspect you used Goerli addresses on Mainnet?

Reply:
thats in the agent, the service has them for mainnet